tests/testthat/_snaps/pkgcheck/checks.md

```diff
@@ -0,0 +1,171 @@
+## Checks for [testpkgchecknotapkg (v0.0.0.9000)]()
+
+git hash: [](/tree/)
+
+- :heavy_check_mark: Package uses 'roxygen2'
+- :heavy_multiplication_x: Package does not have a 'contributing.md' file
+- :heavy_multiplication_x: Package does not have a 'CITATION' file
+- :heavy_multiplication_x: Package does not have a 'codemeta.json' file
+- :heavy_check_mark: All functions have examples
+- :heavy_multiplication_x: Package 'DESCRIPTION' does not have a URL field
+- :heavy_multiplication_x: Package 'DESCRIPTION' does not have a BugReports field
+- :heavy_check_mark: Package name is available
+- :heavy_multiplication_x: Continuous integration checks unavailable (no URL in 'DESCRIPTION')
+- :heavy_multiplication_x: Package coverage is 0% (should be at least 75%)
+- :heavy_multiplication_x: R CMD check found 1error
+- :heavy_check_mark: R CMD check found no warnings
+- :heavy_check_mark: This package still has TODO standards and can not be submitted
+
+**Important:** All failing checks above must be addressed prior to proceeding
+
+Package License: GPL-3
+
+---
+
+## 1. rOpenSci Statistical Standards ([`srr` package](https://github.com/ropensci-review-tools/srr))
+
+This package is in the following category:
+
+- *Regression and Supervised Learning*
+
+:heavy_check_mark: This package still has TODO standards and can not be submitted
+
+Click [here to see the report of author-reported standards compliance of the package with links to associated lines of code](report.html), which can be re-generated locally by running the [`srr_report()` function](https://docs.ropensci.org/srr/reference/srr_report.html) from within a local clone of the repository.
+
+---
+
+
+## 2. Statistical Properties
+
+This package features some noteworthy statistical properties which may need to be clarified by a handling editor prior to progressing.
+
+<details>
+<summary>Details of statistical properties (click to open)</summary>
+<p>
+
+The package has:
+
+- code in C++ (72% in 2 files) and R (28% in 4 files)
+- 1 authors
+- no  vignette
+- no internal data file
+- 1 imported package
+- 1 exported function (median 3 lines of code)
+- 2 non-exported functions in R (median 3 lines of code)
+- 2 R functions (median 5 lines of code)
+
+---
+
+Statistical properties of package structure as distributional percentiles in relation to all current CRAN packages
+The following terminology is used:
+- `loc` = "Lines of Code"
+- `fn` = "function"
+- `exp`/`not_exp` = exported / not exported
+
+The final measure (`fn_call_network_size`) is the total number of calls between functions (in R), or more abstract relationships between code objects in other languages. Values are flagged as "noteworthy" when they lie in the upper or lower 5th percentile.
+
+|measure                 | value| percentile|noteworthy |
+|:-----------------------|-----:|----------:|:----------|
```

Comment:
I am realizing we might want to use some fixed CRAN data for this? Otherwise the percentiles will keep changing?

Reply:
Maybe I should snapshot only part of the report? Is that easy to do?

Reply:
That one I'm not sure about, and will depend partly on upcoming `pkgstats` methods for keeping the data up to date. for the moment it can remain hard-coded, and we can keep #51 open until properly addressed. Okay?

Reply:
Sounds good